[ZAP] Enable Level Control and Color Control Clusters

thomaskang opened the pull request:
New features:

- Support Extended Color Light device type.

Changes:

Fixes:

Issues:

> !!!!!!!!!! Please delete the instructions below and replace with PR description
>
> If you have an issue number, please use a syntax of
> `Fixes #12345` and a brief change description
>
> If you do not have an issue number, please have a good description of
> the problem and the fix. Help the reviewer understand what to expect.
>
> Make sure you delete these instructions (to prove you have read them).
>
> !!!!!!!!!! Instructions end

thomaskang committed:
[ZAP] Enable Level Control and Color Control Clusters

New features:

- Support Extended Color Light device type.

Changes:

Fixes:

Issues:

wade-wu115 committed:
[ZAP] Revise for Color Control/Level Control

Changes:

- Support TC-CC-2.1
- Support TC-IDM-10.4
- Support TC-LVL-2.1

Fixes:

Issues:

wade-wu115 commented:
There are two amends in the commit  [692ee91] (https://github.com/Senscomm/connectedhomeip/pull/19/commits/692ee919f8b4cf01a444715a3a1167df616f5024) we need to pay attention to.

1. Update Cluster On/Off Attribute: FeatureMap value
2. Disable Cluster Identify]-EP1 Commands 0x40 because our PICS do not support I.S.C40.Rsp

Before making these modifications, the On/Off TC-IDM-10.4 EP1 should have met these issues and tested failed. Perhaps we overlooked them during our previous tests.

wade-wu115 commented:
## Verification
### Build
> scripts/build/build_examples.py --target senscomm-scm1612s-light build

> scripts/build/build_examples.py --target senscomm-scm1612s-ayla build

### Run-time

| App | Test | Pass |
| --- | --- | --- |
| light | Commissioning and on/off control <br/> chip-tool | Y <br/> Y |
| ayla | Commissioning and led control by Aura | Y |
| TCs| TC-IDM-10.4 <br/>TC-LVL-2.1 <br/>TC-CC-2.1 <br/> TC-LVL-4.1 <br/> TC-OO-1.1 <br/>TC-OO-2.4| Y <br/> Y <br/> Y <br/> Y <br/> Y <br/> Y |

(Note)

For Ayla-Matter, to properly reset DUT to the factory default such that it will again be able to be commissioned by the Aura app while keeping DSN, OEM etc. in the system partition, we must perform the following sequence:

1. 
> fs umount /;fs format /
2.
>ayla reset factory
(DUT will reboot)
3.
>  ayla setup_mode disable
ayla reset
4.
> Set up via Aura app.

thomaskang committed:
[BUILD] Convert wise-sdk as a public repo.

New features:

Changes:

- Convert wise-sdk to be a public submodule.
- Define system partition info for Ayla.

Fixes:

Issues:
